lib/internal/crypto/keys.js

```diff
@@ -703,6 +703,45 @@ ObjectDefineProperties(CryptoKey.prototype, {
   },
 });
 
+function defineCryptoKeyProperties(
+  key,
+  keyObject,
+  algorithm,
+  extractable,
+  keyUsages,
+) {
+  ObjectDefineProperties(key, {
+    [kKeyObject]: {
+      __proto__: null,
+      value: keyObject,
+      enumerable: false,
+      configurable: false,
+      writable: !keyObject,
```

Comment:
Why does it need to be writable sometimes, and sometimes not? Why not configurable? Can you add comments (or maybe JSDoc for the `defineCryptoKeyProperties` function)

Reply:
> Why does it need to be writable sometimes, and sometimes not?

Because when CryptoKey gets transferred (e.g. via postMessage) then the constructor doesn't have the properties, they're provided by the time kDeserialize is called later in the stack. I can get around this by not calling defineCryptoKeyProperties if keyObject is missing.

JSDoc added.

Reply:
Thanks for the explanation, I think it still makes sense to have it in a comment so future reader can understand why `writable` is not a constant.

Reply:
Would you mind suggesting a comment that is correct? I doubt my understanding of the transferrable objects.

Reply:
Maybe we could forego the comment and set `writable: true`?

Reply:
Similar properties on KeyObject, such as `kHandle` are also not writable, non-writable is IMHO the final desirable state.